cfme/tests/cloud_infra_common/test_html5_vm_console.py

```diff
@@ -191,3 +191,4 @@ def _validate_login():
             assert command_result
     finally:
         vm_console.close_console_window()
+        appliance.server.logout()
```

Comment:
Hey @psav, logout was required because when running the Test back 2 back against RHV and VMware Providers, I would face the following issue:
If I just finished test for RHV, code would proceed to adding VMware Provider and once it is added, when it will navigate to Infrastructure -> Virtual Machines Page, I would get "Page Does not Exists" Error, because the browser will try to go to the VM details page of RHV VM which is already deleted and test would fail.Logging out would get rid of this issue.